src/enums/enums.badges.js

```diff
@@ -0,0 +1,2 @@
+// UPDATE ME: Figure this out.
```

Comment:
@callmekungfu @Wuon @DavidHewWing When do you guys wanna meet to set these? We may as well as do it before we merge this PR

Reply:
Can we do this ASAP

Reply:
Since I have added a way to easily make this possible, we can figure this out later going to continue development.